test_times.py

```diff
@@ -0,0 +1,10 @@
+from times import time_range
+from times import compute_overlap_time
```

Comment:
You can do this in one line (but what you did is correct!)
```suggestion
from times import compute_overlap_time, time_range
```